saleor/graphql/core/scalars.py

```diff
@@ -163,3 +164,19 @@ class Minute(graphene.Int):
 
 class Day(graphene.Int):
     """The `Day` scalar type represents number of days by integer value."""
+
+
+class PositiveInteger(graphene.Int):
```

Comment:
Should we follow the incorrect name convention with this new Scalar? 
By that I mean positive != nonnegative.

Reply:
Why would you suggest? Considering Saleor is following the same naming convention for `PositiveDecimal` or there is a plan to refactor that as well?

Reply:
Consistency > correctness in this case unless we're going to change it consistently across the board.

Reply:
@mirekm Can you give some suggestion then, I have difficultly understanding how is the consistency being violated here? `PositiveInteger` or `PositiveInt`  are names that I can think of.

Reply:
@emZubair My question was open question, not only to you, but other developers to start discussion here. It didn't mean that I disagree with proposed name.

Reply:
@emZubair my comment was in support of your naming as it follows the current convention == consistent.